### PR TITLE
add perm choice for #1922

### PR DESCRIPTION
**Pull request by teamgeode**
PR in ref to #1922

### CODE REVIEW

File: geonode/layers/templates/layers/layer_list.html [hunk 1]
```diff
@@ -21,7 +21,9 @@ <h2 class="page-title">{% trans "Explore Layers" %}</h2>
   {% endwith %}
   {% endwith %}
   {% endwith %}
+  {% with is_layer_list='true' %}
```
Comment (simod) [inline geonode/layers/templates/layers/layer_list.html]: could we just use is_layer here so that we don't have to modify the other file?

**Comment by teamgeode** [comment]
Yes we can.
Will change that if you want. But is_layer_list it's more to make a difference between one layer and a list of layer.

**Comment by simod** [comment]
I would be +1 on just is_layer as from the permissions perspective there's no difference whether is one layer or many layers :)

**Comment by simod** [comment]
@teamgeode are those layer bulk permissions actually working?
The issue is that the edit layer style and layer data permissions are set on Layer objects and not on ResourceBase while the bulk permissions view is only acting on ResourceBase, see here https://github.com/GeoNode/geonode/blob/master/geonode/security/views.py#L102. So I suspect that those two perms are not actually applied (and we don't have tests for those which should be added as well)

**Comment by simod** [comment]
@teamgeode forgive me just noticed now this https://github.com/GeoNode/geonode/blob/master/geonode/security/models.py#L186 that should make everything working :)
Thanks!

**Comment by teamgeode** [comment]
I was tested with some layer.

User_A make layer and no one have perm on it except viewing.
User_B try to modify style but can have the style panel.
User_A add User_B on the style perm list by list layer page.
User_B have access on the style panel.

That’s look working for me.

I will make more test tomorrow and tell you inform.

**Pull request merged**